copyfromstaging.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
Out of interest, would it be feasible to put this into a `.OwlBot` directory or similar? I already think we have too many scripts in the root directory, and would like to move a bunch... if we could avoid creating any more, that would be great. But that's not a blocker.

Reply:
(Or in the same directory as the Dockerfile, of course.)

Reply:
Yes.